manual/cypher/cypher-docs/src/docs/dev/syntax/parameters.asciidoc

```diff
@@ -3,19 +3,32 @@
 
 Cypher supports querying with parameters.
 This means developers don't have to resort to string building to create a query.
-In addition to that, it also makes caching of execution plans much easier for Cypher.
+Additionally, parameters make caching of execution plans much easier for Cypher, thus leading to faster query execution times.
 
-Parameters can be used for literals and expressions in the `WHERE` clause, for the index value in the `START` clause, index queries, and finally for node/relationship ids.
-Parameters can not be used as for property names, relationship types and labels, since these patterns are part of the query structure that is compiled into a query plan.
+Parameters can be used for:
 
-Accepted names for parameters are letters and numbers, and any combination of these.
+* Literals and expressions (such as in the `WHERE` clause)
```

Comment:
For long form list items please terminate each item as if it were an independent sentence.
In general please use on of two formats for lists:
a)
```
This list:
* Has items with more than just one or two words.
* Capitalizes and terminates each item as if it were an independent sentence.
```
b)
```
This list:
* is terse
* not sentence-styled
```
I'd say the first list fits the first format, the second the second.